src/InputField/index.js

```diff
@@ -226,8 +171,8 @@ InputField.propTypes = {
     help: PropTypes.string,
     icon: PropTypes.element,
     trailIcon: PropTypes.element,
-    status: PropTypes.oneOf(['default', 'valid', 'warning', 'error']),
-    size: PropTypes.oneOf(['default', 'dense']),
+    status: iconStatusPropType,
+    size: PropTypes.oneOf([inputSizes.DEFAULT, inputSizes.DENSE]),
```

Comment:
Since you import `inputSizes` already, maybe we could expose a list of the sizes and do something like: `PropTypes.oneOf(inputSizes.alternatives)`?

Any drawbacks?

Reply:
Hmm what would be the usage of said list except for using it for the propTypes?
The user of `ui-core` certainly would only care about the constants.

We could do the following:
```js
import { identity, map } from 'lodash/fp'
const extractValues = map(identity)

InputField.propTypes = {
    // ...
    status: PropTypes.oneOf(extractValues(iconStatuses)),
    size: PropTypes.oneOf(extractValues(inputSizes)),
    // ...
}
```

Then we wouldn't need to adjust all propTypes of components using those sizes.
On the other hand.. this introduces complexity, some components might supports only certain inputSizes and when adding new sizes, the input components would have to be touched anyways in order to support the new size if they should handle the new size.

Or did I overlook another possible use-case for `inputSizes.alternatives`?

Reply:
You're right, verbose and simple is better here.